usr/lib/prime-indicator/prime-indicator

```diff
@@ -259,14 +259,14 @@ class Indicator:
             os.system("budgie-session --logout")
         elif env.startswith("lxqt"):
             os.system("lxqt-leave --logout ")
-        elif env.startswith("gnome") or env.startswith("pantheon") or env.startswith("unity"):
+        elif env.startswith("ubuntu:gnome") or env.startswith("gnome") or env.startswith("pantheon") or env.startswith("unity"):
```

Comment:
```python
elif env.startswith(("ubuntu:gnome", "gnome", "pantheon", "unity")):  # ?
```

Reply:
startswith's signature is: `str.startswith(prefix[, start[, end]])`

Reply:
Totally didn't notice that, haha.